apps/studio/components/interfaces/DiskManagement/fields/ComputeSizeField.tsx

```diff
@@ -195,79 +192,96 @@ export function ComputeSizeField({ form, disabled }: ComputeSizeFieldProps) {
+                                          <span className="text-foreground-light translate-y-[1px]">
+                                            {' '}
+                                            /{' '}
+                                            {compute.price_interval === 'monthly'
+                                              ? 'month'
+                                              : 'hour'}
+                                          </span>
+                                        </>
+                                      )}
+                                    </div>
+                                  </div>
+
+                                  <div className="w-full">
+                                    <div className="px-3 text-sm flex flex-col gap-1">
+                                      <div className="text-foreground-light flex gap-2 items-center">
+                                        <Microchip
+                                          strokeWidth={1}
+                                          size={14}
+                                          className="text-foreground-lighter"
+                                        />
+                                        <span>
+                                          {compute.identifier === 'ci_nano' && 'Up to '}
+                                          {compute.meta?.memory_gb ?? 0} GB memory
+                                        </span>
+                                      </div>
+                                      <div className="text-foreground-light flex gap-2 items-center">
+                                        <CpuIcon
+                                          strokeWidth={1}
+                                          size={14}
+                                          className="text-foreground-lighter"
+                                        />
+                                        <span>
+                                          {compute.meta?.cpu_cores ?? 0}
+                                          {compute.meta?.cpu_cores !== 'Shared' &&
+                                            `-core ${cpuArchitecture}`}{' '}
+                                          CPU
+                                        </span>
+                                      </div>
+                                    </div>
                                   </div>
                                 </div>
                               </div>
-                            </div>
-                          </>
+                            </TooltipTrigger>
+                            {lockedMicroDueToPITR && (
+                              <TooltipContent side="bottom" className="w-64 text-center">
+                                Project has PITR enabled which requires a minimum of Small compute.
+                                Please{' '}
+                                <InlineLink href="/project/_/settings/addons?panel=pitr">
```

Comment:
sorry realised we have the project ref!
```suggestion
                                <InlineLink href={`/project/${project?.ref}/settings/addons?panel=pitr`}>
```